book-a-reading-room-visit.web/wwwroot/js/toggle-reading-room.js

```diff
@@ -1,14 +1,33 @@
 ﻿const roomSelectionForm = document.querySelector('#room-selection-form');
-const updateButton = document.querySelector('#room-selection-form > fieldset > button[type=submit]')
 
 if (roomSelectionForm !== null) {
+    const updateButton = document.querySelector('#room-selection-form > fieldset > button[type=submit]');
+    const dateListContainer = document.querySelector('#date-selection > div');
+    const dateList = document.querySelector('#date-selection > div > ul');
+    const radioButtonsArray = Array.from(document.querySelectorAll('#room-selection-form input[type=radio]'));
+
     const hideUpdateButton = () => {
         updateButton.style.display = 'none';
     }
 
+    const disableButtons = (radioButtonsArray) => {
+        radioButtonsArray.forEach(button => {
+            button.setAttribute('disabled', true);
+        })
+    }
+
     hideUpdateButton();
 
     roomSelectionForm.addEventListener('change', e => {
-        e.currentTarget.submit()
+        const loadingMessageContainer = document.createElement('p');
+
+        loadingMessageContainer.setAttribute('aria-live', 'polite');
```

Comment:
I think it would probably be better to add the `aria-live` attribute to `dateListContainer` so that changes to its child nodes are announced. I'm not sure (and might be wrong here) but I'm not sure that _introducing_ an element which has `aria-live` on it will result in an announcement. I'm just thinking it's probably safest to introduce it to an ancestor element.

One other thing I've noticed is that the MDN page has `role="region"` applied to the element that has `aria-live` probably worth adding this too. 

One final thing - just check with Piriya if the 'Loading...' message is OK.

Reply:
Thanks @gtvj. I've made those changes. Just waiting for the loading message to be approved